### PR TITLE
Create Urdu translation for contents in messages.json.

### PR DESCRIPTION
**Pull request by Toprun123**
I translated the messages.json file to Urdu under the directory `ur`. If there is any mistake please tell me.

### CODE REVIEW

**Comment by Toprun123** [comment]
Please add `hacktoberfest-accepted` tag.

**Comment by rugk** [comment]
Hi @Toprun123,
first of all, thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

Please subscribe to #138 to get notified for new updates on translations etc. :smiley:

BTW note if you want you can also write the Changelog entry already, just copy an existing one [as explained as a side-note here](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#credit).

**Review by rugk — APPROVED** [review]
But LGTM, thanks.
Also you may add yourself to the CONTRIBUTORS.md file and the changelog as a translator to [give credits here](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#credit).

**Comment by Toprun123** [comment]
I hope dto it soon but i started working on another project so it will probably take me some time.

**Comment by rugk** [comment]
Thanks yeah fine! Also notice there are more things to translate like the add-on listings on AMO (addons.mozilla.org) etc. [See the contributing guide for more information.](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translation-of-add-on-listings) Just in case you _want_ to.

File: src/_locales/ur/messages.json [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+    "ariaMessageError": {
+      "message": "مسلہ",
+      "description": "the aria label to label the message box as an error message box"
+    },
+    "ariaMessageWarning": {
+      "message": "انتباہ",
+      "description": "the aria label to label the message box as an warning message box"
+    }
+  }
+
```
Comment (rugk) [inline src/_locales/ur/messages.json:429]:
```suggestion
}
```

FYI as far as I see this is the correct indentation?

**Review by rugk — CHANGES_REQUESTED** [review]

File: src/_locales/ur/messages.json (hunk 1 above)
Comment (Toprun123) [reply src/_locales/ur/messages.json]: I've indented the whole thing half half based on 2 and 4 spaces by mistake.. I've changed it now to follow a 2 space indentation throughout.

**Review by rugk — APPROVED** [review]

**Pull request merged**